### PR TITLE
Create new poll 2024-07-29

### PR DESCRIPTION
**Pull request by CivicSage3**
SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md

### CODE REVIEW

File: governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The community can vote in this poll to express support or opposition to activating the Lido LST yield oracle and the linked Interest Rate Model for the ETH market on SparkLend with the following parameters:
+
+- `base rate`: **0%**
+- `variableRateSlope1Spread`: **0%**
+- CappedFallbackRateSource `upperBound`: **5.5%**
+- CappedFallbackRateSource `lowerBound`: **2%**
+- CappedFallbackRateSource `defaultRate`: **3%**
+- `slope 2`: 120%
+
+Please review the discussion [thread](https://forum.makerdao.com/t/jul-27-2024-proposed-changes-to-spark-for-upcoming-spell/24755) to help inform your position before voting.
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md:39]: Consider adding the contract links provided in the [forum post](https://forum.makerdao.com/t/jul-27-2024-proposed-changes-to-spark-for-upcoming-spell/24755)

File: governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-07-29T16:00:00
+end_date: 2024-08-01T16:00:00
+---
+# Poll: SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md:18]:
```suggestion
# Poll: SparkLend Ethereum - Activate Lido Liquid Staking (LST) Interest Rate Model (IRM) - July 29, 2024
```

File: governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to activating the Lido LST yield oracle and the linked Interest Rate Model for the ETH market on SparkLend with the following parameters:
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md:30]: Abbreviations should be reviewed here. The title introduces LST and IRM as abbreviations. I would suggest they both still be written "Liquid Staking (LST)" and "Interest Rate Model (IRM)". Regardless, the sentence should make sure the same format is used for LST and IRM.

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
Minor changes. The title suggestion is the only blocking item from the review. Please also consider if more resource links should be added, our team supports either adding them in, or leaving as is.

**Comment by MakerJanSky** [comment]
# Poll Review

## Admin I

- [x] Is the poll expected this week?
- [x] Do you understand the question the poll is asking of governance?

## Content

- [x] Does the filename have the .md extension?
- [x] Does the filename contain banned characters? <>:"/\?*|
- [x] Do the titles contain banned characters?  <>:"/\?*
- [x] Are the poll options too long?  (Less than 30 characters as a rough guide)
- [x] Do the options have distinct numbers in the preamble?
- [x] Are the start and end dates present and correct?
- [x] Does the vote type match desired vote type for the poll?
- [x] Is the default option correct in metadata?
- [x] Is the preamble ready for submission?
- [x] Is the vote descriptor present and does it match the vote type?
- [x] Are the 'Review', 'Outcomes', and 'Resources' sections present?
- [x] Will the Data/parameters present in the poll be accurate at the time the poll is live?
- [x] Are the dates correct within the text?
- [ ] Are all the relevant links provided?
- [x] Are all the links working?
- [x] Are the outcomes clear and comprehensive?
- [x] Do any '$' parameter signifiers remain?
- [x] Does the poll summary accurately reflect the poll content?
- [x] Read the poll text in full and highlight any other potential issues.

## Admin II

- [x] Raise any issues with the poll author via review on GitHub.
- [ ] Inform the reviewer when the poll passes review.

File: governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+- `base rate`: **0%**
+- `variableRateSlope1Spread`: **0%**
+- CappedFallbackRateSource `upperBound`: **5.5%**
```
[the remark this answers is not in the document]
Comment (blimpa) [reply governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md:34]: Sorry. Couldn't find documentation on these. Not sure this adds much value. @VoteWizard , @MakerJanSky ?

File: governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to activating the Lido LST yield oracle and the linked Interest Rate Model for the ETH market on SparkLend with the following parameters:
+
+- `base rate`: **0%**
+- `variableRateSlope1Spread`: **0%**
```
[the remark this answers is not in the document]
Comment (blimpa) [reply governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md:33]: We've used the Aave docs for these in the past

**Review by blimpa — CHANGES_REQUESTED** [review]

File: governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (CivicSage3) [reply governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md:34]: I couldn't find anything else either. It does give some basic definition, so I'd say better than nothing, even if it does not provide comprehensive information.

**Review by MakerJanSky — APPROVED** [review]
Nice work, all.

**Pull request merged**